### DISCUSSION TITLE
CANNABIS STRAINS

### DISCUSSION THREAD

**Discussion by cocoath** [openthc/api#24] [General]
Kalamata,city of Greece,are producing a native excellent quality rich strain about 90%sativa between two mountains ,river side,or beach alongside, from 0m and the big,tall,trees,in the same spot hundreds of years produce tones of the best buds in the world in temperate clima,about 45⁰ geo.width.
The nearest situation with the ganja of kalamata is the strain that named Durban poison and located in South Africa at the analogical geo position,but in the south hemisphere of Earth 🌍. The 2 of cannabis species are too similar, familiar and they are a little difficult to find them nowadays, especially in the purest genotype. They're 4m+ tall sativa based strains, native, seedling 🌱 along and making brother plants,so especially river side making often big,many acres, plantations like jungle with perfect situations for cannabis treatment,natural . A special characteristic of the trees are red threads ,deep like blood's color, another character is the psychoactivity of the plant, and it's not a reason of the huge percentage of THC substances. Typically,a strong cannabis smoker, with 2 inhales of this sweet,other times red wine taste or pine tree,or different fruits of the forest becomes so much stoned 😁😲 multiple times from another type of bud, with increased laughing mood,so psychological effects,that sometimes many users fall in bad trip situation, with false illusions, like stalking frenzy.👽🥳👻
In California the climate is nearly,but there's not ever had made a related strain to African Durban poison neither the Greek kalamatiano ganja tree, like the microclimate of two positions making this seed easy and perfectly, natural and native like a wild weed of topic environment.

**Comment by mashiox** [comment]
Thanks for posting!

This subject appears to be off-topic, so we are closing the discussion. If you need more assistance, feel free to open another issue, or email us at help@openthc.com.
